examples/05_flag_args.rs

```diff
@@ -38,6 +47,11 @@ fn main() {
         println!("Awesomeness is turned on");
     }
 
+    // Same thing with `pseudo-flag`
+    if matches.value_of("pseudo-flag") != Some("false") {
+        println!("Pseudo-flag is turned on");
+    }
+
```

Comment:
And this one.

Reply:
Could you please explain why? I copied this directly from https://github.com/clap-rs/clap/issues/1649#issuecomment-661274943.

Reply:
Because this PR is not related to #1649 at all.

Reply:
I'm aware of the fact that this PR is unrelated, although I'm following the advice presented in https://github.com/clap-rs/clap/pull/2619#issuecomment-888433774:

> > The rest of this problem is to somehow make users of this crate more aware of the pseudo-flag approach. Any advice?
> 
> imo we should have a place to document common arg patterns. Unsure if `examples/` is the best place or the website.

Reply:
I am not sure why that was even discussed in the first place. Let's just remove it for now and add it to examples when actually resolving #1649

Reply:
Reverted.